src/deadline/job_attachments/asset_sync.py

```diff
@@ -482,6 +498,8 @@ def sync_inputs(
                     f"Virtual File System not found, falling back to {JobAttachmentsFileSystem.COPIED} for JobAttachmentsFileSystem."
                 )
 
+        # Copied Download flow
+        self._ensure_disk_capacity(session_dir, total_input_size)
```

Comment:
Just at a glance, it looks like we'll always be ensuring disk capacity, even if using VFS. Does the above code make the session dir have infinite space when inspected by `shutil.disk_usage`?

Reply:
I think VFS returns at line479/ 495, starting at 501 is regular JA downloads?

Reply:
As David mentioned, we return early if using VFS. We fallback to COPIED if some settings aren't set correctly for VFS to run.